### DISCUSSION TITLE
Where can I find a complied TIFF library for Windows?

### DISCUSSION THREAD

**Discussion by aous72** [aous72/OpenJPH#49] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyldliu** [comment]
Hi Aous,
Where I can find the TIFF library for Windows build? In the CMakeList.txt
  ```
if (MSVC)
    set(TIFF_PATH "C:\\Program Files\\tiff")
  endif()
```
On my dev machine the tiff folder does not exist, I'm using Visual Studio 2022 v17.8.
Thanks!
David

**Comment by aous72** [comment]
Hi David,

I do not remember how I got it.  I think I compiled it myself. 
The library comes with many external dependencies, only some of which however were included.

Here is what I have in that folder.
[tiff.zip](https://github.com/aous72/OpenJPH/files/14060335/tiff.zip)
See if that helps with the images you are interested in.

Alternatively, you can disable TIFF support.

Let us know how that goes with you.

Kind regards,
Aous.

**Comment by hyldliu** [comment]
Thank you, Aous! 
With the TIFF library, I've been able to build the solution with Visual Studio 2022.
David
